testing/test_pagination.py

```diff
@@ -184,3 +184,7 @@ def test_custom_page(paginator):
     paginator.go_to_page(2)
     assert paginator.current_page == 2
     assert disp_items != paginator.displayed_items
+
+
+def test_pagination_disabled(paginator):
+    assert paginator.is_enabled
```

Comment:
This check only `enabled` paginator. I think, It will be great if you include the `disabled` paginator.